mappings/net/minecraft/client/particle/ParticleExplode.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/bmy
+CLASS none/bmy net/minecraft/client/particle/ParticleExplode
```

Comment:
IIRC, "Explode" is large white smoke particles, while Large/Huge are crescents. Small explosions just use the first, medium ones use Large+Explode, and big ones use all three.

`LargeSmoke`? `ExplosionSmoke`?